### DISCUSSION TITLE
Nightly build with GH actions and draft release?

### DISCUSSION THREAD

**Discussion by princemaple** [spacedriveapp/spacedrive#187] [Feature Request]
It'd be great to be able to simply download and play with it.

**Comment by xPolar** [comment]
This is planned, however, Spacedrive is not ready to have public builds out just yet. Most of the work done so far has just been backend, and a lot of the UI isn't very functional. If you want to mess around with Spacedrive I would recommend compiling it yourself, there should be a guide located in the [Contributing Guide](https://github.com/spacedriveapp/spacedrive/blob/main/CONTRIBUTING.md).

**Reply by princemaple** [reply]
Thanks
